packages/components/src/components/hds/stepper/nav/index.hbs

```diff
@@ -0,0 +1,76 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<div
+  class={{this.classNames}}
+  ...attributes
+  {{style --progress-bar-width=this.progressBarWidthStyle}}
```

Comment:
Perhaps you could set inline styles similarly to how it's done here:

* HBS: https://github.com/hashicorp/design-system/blob/9194be0b1382b1297a772a30d97e5b5c88c8bfec/packages/components/src/components/hds/layout/grid/index.hbs#L10
* TS: https://github.com/hashicorp/design-system/blob/9194be0b1382b1297a772a30d97e5b5c88c8bfec/packages/components/src/components/hds/layout/grid/index.ts#L84